docs/swarm/swarm-tutorial/rolling-update.md

```diff
@@ -67,6 +75,15 @@ applies the update to nodes according to the `UpdateConfig` policy:
     redis
     ```
 
+    The scheduler applies rolling updates as follows:
+
+    1. Schedule the initial batch of tasks.
```

Comment:
Maybe there should be a step before this for stopping the initial batch of tasks that are being replaced.